R/tt_sort.R

```diff
@@ -43,86 +40,67 @@ cont_n_onecol <- function(j) {
-#'   given by [table_structure()].
+#' A list of valid (non-wildcard) paths can be seen in the `path` column of the `data.frame` created by [make_row_df()]
+#' with the `visible_only` argument set to `FALSE`. It can also be inferred from the summary given by
+#' [table_structure()].
 #'
-#'   Note that sorting needs a deeper understanding of table structure in
-#'   `rtables`. Please consider reading related vignette
-#'   ([Sorting and Pruning](https://insightsengineering.github.io/rtables/main/articles/sorting_pruning.html))
-#'   and explore table structure with useful functions like [table_structure()]
-#'   and [row_paths_summary()]. It is also very important to understand the
-#'   difference between "content" rows and "data" rows. The first one analyzes
-#'   and describes the split variable generally and is generated with
-#'   [summarize_row_groups()], while the second one is commonly produced by
-#'   calling one of the various [analyze()] instances.
+#' Note that sorting needs a deeper understanding of table structure in `rtables`. Please consider reading the related
+#' vignette ([Sorting and Pruning](https://insightsengineering.github.io/rtables/main/articles/sorting_pruning.html))
+#' and explore table structure with useful functions like [table_structure()] and [row_paths_summary()]. It is also
+#' very important to understand the difference between "content" rows and "data" rows. The first one analyzes and
+#' describes the split variable generally and is generated with [summarize_row_groups()], while the second one is
+#' commonly produced by calling one of the various [analyze()] instances.
 #'
-#'   Built-in score functions are [cont_n_allcols()] and [cont_n_onecol()].
-#'   They are both working with content rows (coming from [summarize_row_groups()])
-#'   while a custom score function needs to be used on `DataRow`s. Here, some
-#'   useful descriptor and accessor functions (coming from related vignette):
-#'    - [cell_values()] - Retrieves a named list of a `TableRow` or
-#'        `TableTree` object's values.
-#'    - [obj_name()] - Retrieves the name of an object. Note this can differ
-#'      from the label that is displayed (if any is) when printing.
-#'    - [obj_label()] - Retrieves the display label of an object. Note this
-#'      can differ from the name that appears in the path.
-#'    - [content_table()] - Retrieves a `TableTree` object's content table
-#'        (which contains its summary rows).
-#'    - [tree_children()] - Retrieves a `TableTree` object's direct children
-#'      (either subtables, rows or possibly a mix thereof, though that
-#'      should not happen in practice).
+#' Built-in score functions are [cont_n_allcols()] and [cont_n_onecol()]. They are both working with content rows
+#' (coming from [summarize_row_groups()]) while a custom score function needs to be used on `DataRow`s. Here, some
+#' useful descriptor and accessor functions (coming from related vignette):
```

Comment:
this details is one of the best imo, we should do something similar to improve formats @martincadek